lib/screens/home/widget/home_paging_widget.dart

```diff
@@ -0,0 +1,154 @@
+                const Spacer(),
+                _buildSurveyFooter(context, survey, controller),
+              ],
+            ),
+          ),
+        )
+      ],
+    );
+  }
+
+  Widget _buildSurveyHeader(BuildContext context, SurveyUiModel survey) {
+    return Row(
+      children: [
+        Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Text(
+              DateFormat("EEEE, MMMM dd").format(survey.activeAt).toUpperCase(),
+              style: Theme.of(context)
+                  .textTheme
+                  .titleLarge
+                  ?.copyWith(fontSize: 13),
+            ),
+            Text(
+              _daysSince(context, survey.activeAt),
+              style: Theme.of(context).textTheme.titleLarge,
+            )
+          ],
+          //TODO add user avatar
+        ),
+      ],
+    );
+  }
+
+  String _daysSince(BuildContext context, DateTime dateTime) {
+    final dateFormat = DateFormat('yyyy-MM-dd');
+    final DateTime now = DateTime.parse(dateFormat.format(DateTime.now()));
+    final DateTime date = DateTime.parse(dateFormat.format(dateTime));
+    final difference = now.difference(date).inDays;
+    if (difference == 0) {
+      return AppLocalizations.of(context)!.today;
+    } else if (difference == 1) {
+      return AppLocalizations.of(context)!.yesterday;
+    } else {
+      return AppLocalizations.of(context)!.days_ago(difference);
+    }
+  }
+
+  Widget _buildSurveyFooter(
+    BuildContext context,
+    SurveyUiModel survey,
+    PageController controller,
+  ) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Padding(
+          padding: const EdgeInsets.only(bottom: 20),
+          child: SmoothPageIndicator(
+            controller: controller,
+            count: widget.surveys.length,
+            effect: const ScrollingDotsEffect(
+              dotColor: Colors.white24,
+              activeDotColor: Colors.white,
+              dotHeight: 8,
+              dotWidth: 8,
+            ),
+          ),
+        ),
+        Text(survey.title,
+            maxLines: 2,
+            overflow: TextOverflow.ellipsis,
+            style: Theme.of(context).textTheme.titleMedium),
+        Row(
+          children: [
+            Expanded(
+                child: Text(survey.description,
+                    maxLines: 2,
+                    overflow: TextOverflow.ellipsis,
+                    style: Theme.of(context).textTheme.bodyMedium)),
+            Padding(
+                padding: const EdgeInsets.only(left: 20),
+                child: FloatingActionButton(
+                    backgroundColor: Theme.of(context).primaryColor,
+                    onPressed: () => {},
+                    child: const Icon(
+                      Icons.chevron_right,
+                      color: Colors.black,
+                      size: 56 * 0.75, // 75% of fab size
+                    )))
```

Comment:
```suggestion
Padding(
              padding: const EdgeInsets.only(left: 20),
              child: FloatingActionButton(
                backgroundColor: Theme.of(context).primaryColor,
                onPressed: () => {},
                child: const Icon(
                  Icons.chevron_right,
                  color: Colors.black,
                  size: 56 * 0.75, // 75% of fab size
                ),
              ),
            ),
```

Let's update the format a bit. 😂

Reply:
Always forget the `,` 🥲